easybuild/easyconfigs/l/libpng/libpng-1.2.58.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.2.58'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['283f84b0d020589542716ba3c31c5fcf88fb7fd3082a5bbebfc320534c512e1f']
```

Comment:
This is the checksum for the trial version, the checksum for the full version is different, so I'll need to update this...